core/src/software/aws/toolkits/core/credentials/ToolkitCredentialsProvider.kt

```diff
@@ -5,10 +5,9 @@ package software.aws.toolkits.core.credentials
 
 import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider
 
-abstract class ToolkitCredentialsProvider : AwsCredentialsProvider {
+abstract class ToolkitCredentialsIdentifier {
```

Comment:
This the ID of a provider, should it be named something less general like `ToolkitCredentialsProviderIdentifier`. That name is probably too long, maybe a comment would suffice.

Reply:
Ya, I was turned off of that myself due to the length...